tools/alara_output_processing/alara_output_plotting.py

```diff
@@ -113,6 +107,30 @@ def build_color_map(list_of_pivs, cmap_name='Dark2'):
 
     return {lbl: cmap(i % cmap.N) for i, lbl in enumerate(sorted(all_nucs))}
 
+def build_color_map_from_nucs(all_nucs, cmap_name):
```

Comment:
It's not obvious how this is substantially different from `build_color_map_from_pivs`?